pootle/apps/pootle_store/utils.py

```diff
@@ -193,47 +193,18 @@ def add(self, unit, translation, user=None):
 
     def update_unit_on_accept(self, suggestion):
         unit = suggestion.unit
-
-        # Save for later
-        old_state = unit.state
-        old_target = unit.target
-
-        # Update some basic attributes so we can create submissions. Note
-        # these do not conflict with `ScoreLog`'s interests, so it's safe
-        unit.target = suggestion.target
-        if unit.state == FUZZY:
-            unit.state = TRANSLATED
-        unit.submitted_by = suggestion.user
```

Comment:
its passed as an argument - and the tests (hopefully) check that its correctly set